Tests/Functional/ContentTypes/MenuRecentlyUpdatedPagesElementTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the "headless" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ *
+ * (c) 2021
+ */
+
+declare(strict_types=1);
+
+use FriendsOfTYPO3\Headless\Test\Functional\ContentTypes\BaseContentTypeTest;
+use TYPO3\TestingFramework\Core\Functional\Framework\Frontend\InternalRequest;
+
+class MenuRecentlyUpdatedPagesElementTest extends BaseContentTypeTest
+{
+    public function testMenuContentElement()
+    {
+        $response = $this->executeFrontendRequest(
+            new InternalRequest('https://website.local/')
+        );
+
+        self::assertEquals(200, $response->getStatusCode());
+
+        $fullTree = json_decode((string)$response->getBody(), true);
+
+        $contentElement = $fullTree['content']['colPos1'][7];
+
+        $this->checkDefaultContentFields($contentElement, 16, 1, 'menu_recently_updated', 1);
+        $this->checkAppearanceFields($contentElement, 'default', 'default', 'SpaceBefore', 'SpaceAfter');
+        $this->checkHeaderFields($contentElement, 'Header', 'SubHeader', 0, 2);
+
+        self::assertTrue(is_array($contentElement['appearance']));
+        self::assertTrue(is_array($contentElement['content']));
+        self::assertTrue(is_array($contentElement['content']['menu']));
```

Comment:
don't we need to check for any results here? Can we at least check if there is one recently updated page?